helm/postgres/values.yaml

```diff
@@ -59,7 +59,7 @@ postgresVersion: 14
 
 # imageExporter is the image name for the exporter used as a part of monitoring.
 # This defaults to the value below.
-# imageExporter: registry.developers.crunchydata.com/crunchydata/crunchy-postgres-exporter:ubi8-5.3.0-0
+# imageExporter: registry.developers.crunchydata.com/crunchydata/crunchy-postgres-exporter:ubi8-5.3.1-0
```

Comment:
:wrench: 
There are component versions above for postgres, pgbackrest and pgbouncer that need to be adjusted.

Reply:
Thanks for catching that!